### PR TITLE
Some stuff that needs doing & might be useful

### PR DESCRIPTION
**Pull request by ldir-EDB0**

### CODE REVIEW

**Comment by diizzyy** [comment]
Please don't do a pull request touching multiple things all over the place without any explanition whatsoever.

Regarding cake:
https://github.com/openwrt/openwrt/commit/5c094ff660fd8e750ed1c1f38719a45065e77348#diff-e8d43e2763870232aa8e87e3e734ae55
If this commit breakes OpenWrt then it should fixed in the same tree

As for the rest of the packages, a single PR per package and follow the template.
Is anything compiled/runtime tested?

net/https-dns-proxy:
Why is ```PKG_LICENSE``` removed?

**Comment by hnyman** [comment]
Like diizzyy, I would prefer to keep cake things in one tree. 

I would prefer to have the in-kernel-tree cake module in the OpenWrt tree to depend on kernel 4.19, and respectively have the out-of-tree cake version added back to the OpenWrt tree with dependency for NOT being on kernel 4.19.

That would keep things in the same source tree, and would likely keep things simpler in future.


Btw, buildbot currently builds kmod-sched-cake .ipk also for targets on 4.14, but the .ipk is practically empty. Have you tested that your solution of kmod-sched-cake-oot actually works, and kmod-sched-cake-oot gets picked up by opkg when installing SQM (instead of opkg picking up the empty kmod-sched-cake from the main target ?  (My gut feeling says that removing the in-tree kmod-sched-cake would be needed in any case, but I haven't tested your proposal, so this is guestimate.)

**Comment by ldir-EDB0** [comment]
As per the standard, none of it has seen a compiler and definitely none of it is running on an APU2.

File: net/https-dns-proxy/Makefile [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_MIRROR_HASH:=24b7e4238c37e646f33eee3a374f6b7beb5c167b9c5008cc13b51e5f1f3a44ea
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=bea68401330e611f6e9b75cec84e2dc4e81e52de
+PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
+PKG_SOURCE_DATE:=2019-07-25
+PKG_SOURCE_VERSION:=69e18ea256c9586ea04ef111115fbf20e4b67c96
+PKG_MIRROR_HASH:=6fe67bbce1969ff4b29836670bc5bbed2139ec04a82e37bb6bb6f5ae1c5c3c01
```
Comment (neheb) [inline net/https-dns-proxy/Makefile]: Space after PKG_MIRROR_HASH please.

File: net/https-dns-proxy/Makefile [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-PKG_RELEASE=3
+PKG_RELEASE=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_MIRROR_HASH:=24b7e4238c37e646f33eee3a374f6b7beb5c167b9c5008cc13b51e5f1f3a44ea
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=bea68401330e611f6e9b75cec84e2dc4e81e52de
+PKG_SOURCE_URL:=https://github.com/aarond10/https_dns_proxy/
```
Comment (neheb) [inline net/https-dns-proxy/Makefile]: The / at the end is wrong and can be removed.

File: kernel/kmod-sched-cake-oot/Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2016 LEDE
```
Comment (neheb) [inline kernel/kmod-sched-cake-oot/Makefile]: This should be updated, no?

**Comment by neheb** [comment]
I would like that these be split up in separate pull requests. One per package. Stubby should also get a PKG_RELEASE bump.

**Pull request closed without merging**